src/Zodiac-Tests/ZdcAbstractSocketStreamTest.class.st

```diff
@@ -6,6 +6,9 @@ This is an abstract class, subclasses should implement #socketStreamClass
 Class {
 	#name : 'ZdcAbstractSocketStreamTest',
 	#superclass : 'TestCase',
+	#instVars : [
+		'port'
```

Comment:
There is a release test that is failing because `port` i.v. shadows param of 5 methods.
The easiest fix would be to rename `port` to `socketPort`.